test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -211,7 +211,7 @@ def setUp(self):
 #PBS -joe
 echo $PBS_JOBID
 cpuset_base=`grep cgroup /proc/mounts | grep cpuset | cut -d' ' -f2 | \
-             tr " " "\n" | sed -n '1p'`
+             grep cgroup`
```

Comment:
Isn't the last command supposed to be "head -1" in case there were multiple mount points? You already grep'ed for cgroup.